integration-tests/jpa-oracle/src/main/resources/application.properties

```diff
@@ -3,6 +3,7 @@ quarkus.datasource.username=SYSTEM
 quarkus.datasource.password=hibernate_orm_test
 quarkus.datasource.jdbc.url=${oracledb.url}
 quarkus.datasource.jdbc.max-size=1
+quarkus.datasource.jdbc.transactions=xa
```

Comment:
This is nice, but... it means we're no longer testing the non-XA case with Oracle.

Should we duplicate all such test modules? :/

Reply:
I thought the second datasource would be fine? But maybe it doesn't connect actually.

I can drop this part or add have 2 datasources that point to the same database but one xa and one non-xa?

Reply:
Given we will push more people towards XA, I think it's probably better to have separate tests for XA for each driver we support. It has a cost but I don't feel comfortable not paying it given the new situation with Agroal/Narayana.

WDYT?

Reply:
My concerns are mostly about native images, where the simple fact of using of XA could have a dramatic impact on what GraalVM generates.

I agree duplicating tests would be the way forward.

This will mean even more copy-pasting in tests that already contain a lot of that, but until we find a way to run the same integration test multiple times with different configuration *and* Maven dependencies... that will have to do.

Reply:
I dropped the test change for now.
Will see if it makes sense to have so many new native modules later.